docs/MachineConfiguration.md

```diff
@@ -94,6 +95,10 @@ Ignition config keys as well.
 
 This extends the host's kernel arguments.  Use this for e.g. [nosmt](https://access.redhat.com/solutions/rhel-smt).
 
+### FIPS
+
+This allows to enable/disable [FIPS mode](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/security_guide/chap-federal_standards_and_regulations).
```

Comment:
Can you add a comment here about `if any of the config has FIPS enabled, it'll be set` as well?